packages/codegen-openapi/README.md

```diff
@@ -0,0 +1,55 @@
+# 🏛️ Hyper Fetch Openapi Request Generator 
```

Comment:
Can we pick matching emoji?

Reply:
... I am afraid to ask what would be a matching emoji but let's give it a go.

Reply:
I think wand will be magical enough🪄